### DISCUSSION TITLE
Is it possible to display multiple sequence as single track in remotion studio?

### DISCUSSION THREAD

**Discussion by hikrr** [remotion-dev/remotion#3111] [Q&A]
Hi~

I was trying to create .srt visualisation video, which is currently implemented using `Sequence` and its `from` and `durationInFrames`

however, this will make the remotion studio look like this:
![image](https://github.com/remotion-dev/remotion/assets/80665253/7cab25e3-7ced-49bd-94b6-5ff270de026a)

is it possible to make them appear on a single track?
something like this:
![image](https://github.com/remotion-dev/remotion/assets/80665253/eb701447-8b8a-4e01-81df-b2d25461585d)

or there's some better implementation rather than using `Sequence`

Thanks for your effort and help!

**Comment by JonnyBurger** [comment]
The `<Sequence>`'s work more like PhotoShop layers, so they are underneath each other.
This is the current model and therefore the visualization is correct.

But you are making a good point, we should think about this model and how to prevent showing a waterfall in the future.

**Reply by hikrr** [reply]
Got it. Thanks for your quick response!

**Comment by mnlbox** [comment]
I guess `<Series>` and `<Series.Sequence>` components are better choices for this use-case.
@JonnyBurger Am I wrong? 🤔

**Reply by shahbaz-khan-hub** [reply]
No, you are right, but see, here is an editor, so for something like I want to show different text, elements, etc at the same frame, then I suggest that to use `<Sequence/>` 

- `<Sequence from=...>` in a normal container (like <AbsoluteFill>) is the usual approach for concurrency.

- `<Series>` is specifically for playing multiple items in a row
